bot/code_coverage_bot/utils.py

```diff
@@ -17,21 +16,6 @@ class RunException(Exception):
     """
```

Comment:
You can remove `RunException` at this point, and just use `Exception` in `run_check`